### PR TITLE
[Mac] When tabbing scroll focused control into view.

### PR DESCRIPTION
**Pull request by CartBlanche**
Fixes https://github.com/xamarin/Xamarin.PropertyEditing/issues/466

### CODE REVIEW

File: Xamarin.PropertyEditing.Mac/Controls/Custom/PropertyTextField.cs [hunk 1]
```diff
@@ -11,5 +11,11 @@ public PropertyTextField ()
 			Cell.LineBreakMode = NSLineBreakMode.TruncatingTail;
 			Cell.UsesSingleLineMode = true;
 		}
+
+		public override bool BecomeFirstResponder ()
+		{
+			ScrollRectToVisible (this.Bounds);
```
Comment (ermau) [inline Xamarin.PropertyEditing.Mac/Controls/Custom/PropertyTextField.cs]: We don't need `this` here and we should only actually scroll if `base.Become..` returns true.

Comment (CartBlanche) [reply Xamarin.PropertyEditing.Mac/Controls/Custom/PropertyTextField.cs]: From testing `BecomeFirstResponder ()` is only called when the control is about to receive focus. What scenarios have you seen where `BecomeFirstResponder ()` is called and our property controls should *not* scroll into view? @alanmcgovern Any insights here? I'm happy to make the change, but want to understand exactly why the check is absolutely necessary.

Comment (ermau) [reply Xamarin.PropertyEditing.Mac/Controls/Custom/PropertyTextField.cs]: If you read the docs on `BecomeFirstResponder`, you'll see that the return value is whether or not it actually became first responder. If it hasn't actually become first responder, we don't want to scroll to it. It doesn't matter whether we've seen it happen or not, implementing it correctly is a defensive measure.

Comment (CartBlanche) [reply Xamarin.PropertyEditing.Mac/Controls/Custom/PropertyTextField.cs]: I had already changed it, but hadn't pushed. Done.

**Comment by CartBlanche** [comment]
I had a change for Bool/Combinable, but had not pushed it. I was going to ask about Object/Type though. So I'll just make those accept keyboard focus and that's it then.

**Comment by CartBlanche** [comment]
All controls are now focusable. But I'm now investigating ghost focusing of Brush control, when it is ReadOnly.

**Comment by CartBlanche** [comment]
Ok, so ensuring CanBecomeKeyView is linked to Enabled give the correct behaviour.

**Pull request merged**